assets/css/styles.css

```diff
@@ -4,29 +4,167 @@
   margin: 0;
   padding: 0;
   font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
 }
 
-.topHeader {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding: 3% 6.3% 0 6.2%;
-  box-sizing: border-box;
+html {
+  scroll-behavior: smooth;
+}
+
+header {
+  background-color: white;
 }
 
 .logo {
+  width: 50%;
+  height: 100%;
   font-style: normal;
   font-weight: 700;
-  font-size: 1.125rem;
-  line-height: 1.25rem;
-  color: #6070ff;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  background-color: #fff;
+  padding: 2px;
+}
+
+a {
+  color: #344563;
+  text-decoration: none;
 }
 
 .logo a {
+  margin-left: 24px;
+  background-color: #fff;
+  color: #5464f8;
   text-decoration: none;
-  color: #6070ff;
 }
 
+li {
+  list-style: none;
+}
+
+.nav-bar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #5464f8;
+}
+
+.nav-bar .bar {
+  background-color: #5464f8;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  margin-right: 50px;
+}
+
+.nav-branding {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #5464f8;
+  margin-left: 50px;
+}
+
+.nav-link {
+  transition: 0.7s ease;
+}
+
+.nav-link:hover {
+  color: #5464f8;
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+@media (max-width: 768px) {
+  .hamburger {
+    display: block;
+  }
+
+  .bar {
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: white;
+  }
+
+  .nav-bar.active {
+    background-color: #6070ff;
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 0;
+    gap: 0;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    text-align: left;
+    transition: 0.3s;
+    background-color: #6070ff;
```

Comment:
- [ OPTIONAL ] I think it would be great if you fix the menu to not scroll while the user open it since this is bad for user experience so please delete the scroll you can add this property `overflow-x: hidden;` by this you will avoid the scrolling I am assigning as optional because the previous reviewer didn't mention it but it still v ery important to add it 💯💪🏻✅

Reply:
Thanks, dear @Reem-lab I just finished the changes.

A great hug,

Javier.-